resources/assets/js/src/store/actions/auth.ts

```diff
@@ -55,25 +55,26 @@ export const auth = (email, password) => {
         axios.post(process.env.MIX_OAUTH_ADD, data)
             .then((response) => {
                 const accessToken = response.data.access_token;
-                const expirationDate = new Date(new Date().getTime() + response.data.expires_in * 1000);
+                const expirationDate = new Date(new Date().getTime() + response.data.expires_in);
+                const expiresIn = response.data.expires_in;
                 const headerData = { Authorization: `Bearer ${accessToken}` };
-                dispatch(authGetUserInfo(headerData, accessToken, expirationDate));
+                dispatch(authGetUserInfo(headerData, accessToken, expirationDate, expiresIn));
```

Comment:
why there is expirationDate and expiresIn? can we use only either expirationDate or expiresIn?